### PR TITLE
fix: banner styles for 1%

### PR DESCRIPTION
**Pull request by nickfranciosi**
removes action buttons
updates text
resolves issue where banner could get added twice

### CODE REVIEW

File: src/components/beta_banner/_beta_banner.scss [hunk 1]
```diff
@@ -3,51 +3,16 @@
 .lp-beta-banner {
   background: $beta-banner-color;
   color: $color-white;
-  font-size: 1.6rem;
-  padding-bottom: 1.6rem;
-  padding-top: 1.6rem;
+  font-size: 1.4rem;
+  padding-bottom: 14px;
+  padding-top: 14px;
```
Comment (thomasthesecond) [inline src/components/beta_banner/_beta_banner.scss]: Should the padding values be rem?

File: src/components/beta_banner/_beta_banner.scss [hunk 2]
```diff
@@ -3,51 +3,16 @@
...
-    }
-
-    a:focus {
-      outline: 1px dotted rgba($color-white, .4);
-      outline-offset: 2px;
-    }
+    font-size: 1.1rem;
+    letter-spacing: -.65px;
+    padding-bottom: 8px;
+    padding-top: 8px;
```
Comment (thomasthesecond) [inline src/components/beta_banner/_beta_banner.scss]: Should the padding values be rem here too?

Comment (nickfranciosi) [inline src/components/beta_banner/_beta_banner.scss]: @thomasthesecond I have no idea, I feel like I hear something different every time I change the css in this project

Comment (thomasthesecond) [inline src/components/beta_banner/_beta_banner.scss]: Ok..it's probably fine as pixels 👍

**Comment by nickfranciosi** [comment]
@thomasthesecond is that something new? I never added that

**Comment by thomasthesecond** [comment]
No it's always been that way for global header. I added those styles a couple of weeks ago when @laserchicken noticed the problem.

Solution would be to use px in styles here, but we would have to remove those override styles.

**Comment by thomasthesecond** [comment]
I can update those styles if you want

**Comment by nickfranciosi** [comment]
I will update. Thanks

**Pull request merged**